docs/kyma/docs/04-05-use-your-own-image.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Use your own image
+type: Installation
+---
+
+You can use your own image to prepare the configuration file when installing Kyma on a GKE or AKS cluster.
+
+>**NOTE:** Follow these steps both for your own and the `xip.io` default domain.
+
+1. Check out [kyma-project](https://github.com/kyma-project/kyma) and enter the root folder.
+
+2. Build an image that is based on the current Installer image and includes the current installation and resources charts. Run:
+
+    ```
+    docker build -t kyma-installer:latest -f tools/kyma-installer/kyma.Dockerfile .
+    ```
+
+3. Push the image to your Docker Hub. Run:
+    ```
+    docker tag kyma-installer:latest {YOUR_DOCKER_LOGIN}/kyma-installer:latest
+    docker push {YOUR_DOCKER_LOGIN}/kyma-installer:latest
+    ```
+
+4. Prepare the deployment file.
+
+<div tabs>
+  <details>
+  <summary>
+  GKE
+  </summary>
+
+
+    - Run this command if you use the `xip.io` default domain:
+    ```
+    (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
+
+    - Run this command if you use your own domain:
+    ```
+    (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__DOMAIN__/$DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, follow these steps to prepare the deployment file.
+
+    - Run this command if you use the xip.io default domain:
+    ```
+    (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__.*__//g" > my-kyma.yaml
```

Comment:
check if it is up to date I haven't found anything like PROMTAIL_CONFIG_NAME

Reply:
this content requires review of domain expert and these steps have to simply be reproduced to make sure they work. You can probably also talk with @PavelPavlov46 that I think already played with installation on cluster basend on custom Installer image

Reply:
As discussed, that was due to the different GKE version used.